google-client-loader.html

```diff
@@ -44,32 +39,25 @@
 <script>
   (function() {
     'use strict';
-
     // Stores whether the API client is done loading.
     var _clientLoaded = false;
```

Comment:
can you update the doc string to signify that this is a flag across instances? It's now a bit confusing that there is this static prop and an instance prop that are named the same.